config/default/GUIDE.md

```diff
@@ -0,0 +1,104 @@
+# Reference configuration changes cookbook
+
+These instructions are aimed at developers and technical people who are exploring the reference configuration and are looking to make small changes and explore what is possible. They are necessarily hands-off, as explaining in detail everything that can be accompilshed is a larger task suited to more detailed documentation.
```

Comment:
I think it would help to be more concise here, and remove the next paragraphs.

```suggestion
These instructions are aimed at developers and technical people who are exploring the reference configuration and are looking to make small changes to see what is possible. Detailed tutorials on how to build apps from scratch with the Core Framework will be available soon. In the meantime the best source for detailed documentation is available in the [Developing Community Health Aapplications](https://github.com/medic/medic-docs/blob/master/configuration/developing-community-health-applications.md) page. 
```